scripts/build-docs/build-docs.sh

```diff
@@ -0,0 +1,3 @@
+cargo doc
```

Comment:
can we only run this script if the lib.rs files in the relevant pallets are changed pls?

Reply:
i don't want this to run during every merge

Reply:
So code changes in runtime and node won't trigger uploading docs?

Reply:
code changes will do so, but only for pallet-*/src/lib.rs